modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1910,4 +2005,231 @@ BiFunction<List<Peer>, Long, CompletableFuture<Void>> movePartition(Supplier<Raf
     private <T extends ConfigurationProperty<?>> T directProxy(T property) {
         return getMetadataLocallyOnly ? property : ConfigurationUtil.directProxy(property);
     }
+
+    private Supplier<List<TableIndex>> activeIndexes(UUID tableId, int partId) {
```

Comment:
Why don't supplier return a map indexUid->TableIndex ?

Reply:
for current task it looks enough to return a list only